packages/logger/src/Logger.ts

```diff
@@ -766,6 +782,8 @@ class Logger extends Utility implements ClassThatLogs {
     } = options;
 
     this.setEnvVarsService();
+    // order is important, it uses EngVarsService()
```

Comment:
```suggestion
    // order is important, it uses EnvVarsService()
```

Reply:
Why is the order important here?

Reply:
I wanted to make the order of methods alphabetically sorted, like the linting rule for methods. But `setConsole` uses `setEnvVarsService`, so it won't work.

Reply:
Oh I see, makes sense! Thanks.

Let's correct the typo in the comment (see suggestion) and we can resolve this conversation.